### ISSUE TITLE
Provide additional options for the carousel

### ISSUE THREAD

**Issue by Ash-Matter** [BuilderIO/builder#455]
Its known that the carousel is powered by Slick slider, as stated under the advanced settings in the UI for the carousel.
It would be great to have some more of the options that come with the plugin to be available for configuration in the UI.

For example:

- The ability to toggle the arrows off, just as you have with the dots
- The ability to configure the carousel to fade instead of slide
- The ability to configure the pause and speed of the transition
- Basically, allow any other available configurations to be modified.

**Comment by steve8708** [comment]
Hey @Ash-Matter - these are great suggestions. If you have the bandwidth to, you are more than welcome to send a PR with these updates, these should be straightforward to apply and easily be backwards compatible

Definitely will consider for our roadmap as well, but contributions are more than welcome to speed things up to get what you need!
